shared/common-adapters/user-proofs.native.js

```diff
@@ -1,32 +1,115 @@
+import {checking as proofChecking} from '../constants/tracker'
+
+import type {Props, Proof} from './user-proofs'
+
 export default class ProofsRender extends Component {
+  props: Props;
+
+  _ensureUrlProtocal (url: string): string {
+    return url && (url.indexOf('://') === -1 ? 'http://' : '') + url
+  }
+
+  _onClickProof (proof: Proof): void {
+    if (proof.state !== proofChecking && proof.humanUrl) {
+      openUrl(this._ensureUrlProtocal(proof.humanUrl))
+    }
+  }
+
+  _onClickProfile (proof: Proof): void {
+    if (proof.state !== proofChecking && proof.profileUrl) {
+      openUrl(this._ensureUrlProtocal(proof.profileUrl))
+    }
+  }
+
+  _renderProofRow (proof: Proof, idx: number) {
+    const metaBackgroundColor = shared.metaColor(proof)
+    const proofStatusIconType = shared.proofStatusIcon(proof)
+    const proofNameStyle = shared.proofNameStyle(proof)
+    const onClickProfile = () => { this._onClickProfile(proof) }
+
+    const meta = proof.meta && proof.meta !== metaNone && <Meta title={proof.meta} style={{backgroundColor: metaBackgroundColor}} />
+
+    const proofIcon = proofStatusIconType && <Icon type={proofStatusIconType} style={styles.statusIcon} onClick={() => this._onClickProof(proof)} />
+
+    return (
+      <View style={{...styles.row, paddingTop: idx > 0 ? 8 : 0}} key={`${proof.id}${proof.type}`}>
+        <Icon style={styles.service} type={shared.iconNameForProof(proof)} title={proof.type} onClick={onClickProfile} />
+        <View style={styles.proofNameSection}>
+          <View style={styles.proofNameLabelContainer}>
+            <Text inline className='hover-underline-container' type='Body' onPress={onClickProfile} style={styles.proofName}>
+              <Text inline type='Body' className='underline' style={proofNameStyle}>{proof.name}</Text>
+              <Text className='no-underline' inline type='Body' style={styles.proofType}>@{proof.type}</Text>
+            </Text>
+            {meta}
+          </View>
+        </View>
+        {proofIcon}
+      </View>
+    )
+  }
+
   render () {
     return (
-      <View style={{backgroundColor: 'green', display: 'flex', flex: 1, flexDirection: 'column', overflowY: 'auto'}}>
-        <Text>Proof1</Text>
-        <Text>Proof2</Text>
-        <Text>Proof3</Text>
-        <Text>Proof4</Text>
-        <Text>Proof5</Text>
-        <Text>Proof6</Text>
-        <Text>Proof7</Text>
-        <Text>Proof8</Text>
-        <Text>Proof9</Text>
-        <Text>Proof10</Text>
-        <Text>Proof11</Text>
-        <Text>Proof12</Text>
-        <Text>Proof13</Text>
-        <Text>Proof14</Text>
-        <Text>Proof15</Text>
-        <Text>Proof16</Text>
-        <Text>Proof17</Text>
-        <Text>Proof18</Text>
-        <Text>Proof19</Text>
-        <Text>Proof20</Text>
-        <Text>Proof21</Text>
+      <View style={styles.container}>
+        {this.props.proofs.map((p, idx) => this._renderProofRow(p, idx))}
       </View>
     )
   }
 }
+
+const styles = {
```

Comment:
Reason for changing to the way mentioned by Marco: ESLint will pick up if there are unused styles if the styles are top level variables instead of nested inside an object.